lib/util.js

```diff
@@ -9,4 +9,21 @@ function roundUp(num, precision) {
   return Math.ceil(num * precision) / precision;
 }
 
-module.exports = { roundUp };
+/**
+ * @param {String} vector The inicial vector CVSS
+ *
+ * @return The object containing all separate CVSS values 
+ */
+function getVectorObject(vector) {
```

Comment:
It's a good idea to move that into a function. This change is coming with #22, though. Could you maybe remove the function and move the implementation directly into the `getTemporalScore()` function to prevent merge conflicts? When #22 is merged we can swap the implementation with the function.

Reply:
of course, this is because I refactored the code before the issue was created, I will change